assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js

```diff
@@ -16,6 +16,7 @@ export function postLoadFormat(data: DiscussionPreferencesQueryQuery): Personali
   const { discussionPreferences } = data;
   return {
     title: discussionPreferences && discussionPreferences.tabTitle,
-    favicon: (discussionPreferences && discussionPreferences.favicon) || null
+    favicon: (discussionPreferences && discussionPreferences.favicon) || null,
```

Comment:
I would destructure discussionPreferences above the return ```const { tabTitle, favicon, logo) = data.discussionPreferences;```
and below write each property like this ```title: tabTitle || null;```
current syntax is a bit odd (due to linter)

Reply:
#oui

Reply:
ça génère une erreur flow, je garde l'ancienne syntaxe